adafruit_displayio_layout/layouts/grid_layout.py

```diff
@@ -382,8 +377,12 @@ def _layout_cells(self):
                         self.append(line_obj["tilegrid"])
 
     def add_content(
-        self, cell_content, grid_position, cell_size, cell_anchor_point=None
-    ):
+        self,
+        cell_content: displayio.Group,
+        grid_position: Tuple[int, int],
+        cell_size: Tuple[int, int],
+        cell_anchor_point: Union[Tuple[float, ...], None] = None,
```

Comment:
Here for `cell_anchor_point` I do think `Optional` may be a better choice for the type to convey that it can be None, since there is only 1 other valid type we can also get rid of Union.
```
cell_anchor_point: Optional[Tuple[float, ...]] = None,
```